File: news-cast-explorer-common/src/main/java/org/jesperancinha/newscast/repository/MessageRepository.java
package org.jesperancinha.newscast.repository;

import org.jesperancinha.newscast.model.explorer.Message;
import org.springframework.data.jpa.repository.JpaRepository;

public interface MessageRepository extends JpaRepository<Message, Long> {

    Message findFirstByNewscastMessageIdAndAuthorId(String messagrId, Long authorId);
}

File: news-cast-explorer-cdc/src/main/kotlin/org/jesperancinha/newscast/cdc/repository/MessageRepository.kt
package org.jesperancinha.newscast.cdc.repository

import org.jesperancinha.newscast.cdc.domain.Message
import org.springframework.data.jpa.repository.JpaRepository

/**
 * Created by jofisaes on 08/10/2021
 */
interface MessageRepository :JpaRepository<Message, Long>